libs/domain/product/src/services/adapter/default-product.adapter.ts

```diff
@@ -29,11 +29,23 @@ export class DefaultProductAdapter implements ProductAdapter {
       ...(include ?? []),
     ].filter((type, index, arr) => arr.indexOf(type) === index);
 
+    const categoryNodeFields = [
+      ApiProductModel.CategoryNodeFields.MetaDescription,
+      ApiProductModel.CategoryNodeFields.NodeId,
+      ApiProductModel.CategoryNodeFields.Order,
+      ApiProductModel.CategoryNodeFields.Name,
+      ApiProductModel.CategoryNodeFields.Parents,
+      ApiProductModel.CategoryNodeFields.IsActive,
+    ];
+
     return this.http
       .get<ApiProductModel.Response>(
         `${this.SCOS_BASE_URL}/${this.productEndpoint}/${sku}${
           include ? '?include=' : ''
-        }${include?.join(',') || ''}`
+        }${include?.join(',') || ''}
+        &fields[${
+          ApiProductModel.Includes.CategoryNodes
+        }]=${categoryNodeFields.join(',')}`
```

Comment:
Potential BC on the consumption of this data. Let's assume this method was customized, and it would not return catergory nodes, would the normalizers break? Or still work?
To ensure a non BC DX, the code should ideally fail with a warning. 

We could consider to add the additional fields only when feature flag is on.